content/uiux/concepts/crazy-eights/crazy-eights.md

```diff
@@ -0,0 +1,13 @@
+---
+Title: 'Crazy Eights'
+Description: 'A sketching exercise that challenges people to sketch eight distinct ideas in eight minutes'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Crazy Eights's** is a core Design Sprint method. It is a fast sketching exercise that challenges people to sketch eight distinct ideas in eight minutes. The goal is to push beyond your first idea, which is frequently the basic(least innovative), and to generate a wide variety of solutions to your design(challenge).
```

Comment:
This seems to be taken directly from here:
https://designsprintkit.withgoogle.com/methodology/phase3-sketch/crazy-8s

Reply:
nah I didn't get that information from, seems like many sources have the same verbiage on the topic. No worries ill take core of it.